python/src/BoxCoxFactory_doc.i.in

```diff
@@ -72,16 +76,26 @@ covarianceModel : :class:`~openturns.CovarianceModel`
     Covariance model.
     Should have input dimension equal to input sample's dimension and dimension equal to output sample's dimension.
     See note for some particular applications.
-generalLinearModelResult : :class:`~openturns.GeneralLinearModelResult`
-    Empty structure that contains results of general linear model algorithm.
 
 Returns
 -------
 myBoxCoxTransform : :class:`~openturns.BoxCoxTransform`
     The estimated Box Cox transformation.
+likelihoodGraph : :class:`~openturns.Graph`
+    An empty graph that is fulfilled later with the log-likelihood of the mapped variables with respect to the :math:`lambda` parameter for each component.
+generalLinearModelResult : :class:`~openturns.GeneralLinearModelResult`
+    Empty structure that contains results of general linear model algorithm.
+linearModelResult : :class:`~openturns.LinearModelResult`
+    Empty structure that contains results of linear model algorithm.
 
```

Comment:
the different build functions give different results (not all, maybe mention this above in the "Available usages", maybe something like:
```build(*inputSample, outputSample, covarianceModel, shift*) -> BoxCoxTransform, Graph```
also, now they are not "empty" classes anymore that you need to pass as input

Reply:
indeed, thanks